q2cli/tools.py

```diff
@@ -37,6 +37,21 @@ def export_data(path, output_dir):
     result.export_data(output_dir)
 
 
+def show_importable_types(ctx, param, value):
+    import qiime2.sdk
+
+    PluginManager = qiime2.sdk.PluginManager()
+    set_of_importable_types = PluginManager.importable_types
+    set_of_importable_types = [repr(t) for t in set_of_importable_types]
+    set_of_importable_types = sorted(set_of_importable_types)
+
+    if not value or ctx.resilient_parsing:
+        return
+    for name in set_of_importable_types:
```

Comment:
Can you handle the case where there are no importable types? It'd be useful to let users know, printing something like "There are no importable types in the current deployment."